Scalar/CommandLine/ScalarVerb.cs

```diff
@@ -164,19 +164,9 @@ public virtual void InitializeDefaultParameterValues()
 
         protected GitAuthentication.Result TryAuthenticate(ITracer tracer, ScalarEnlistment enlistment, out string authErrorMessage)
         {
-            string authError = null;
-
-            GitAuthentication.Result result = GitAuthentication.Result.UnableToDetermine;
-            bool runResult = this.ShowStatusWhileRunning(
-                () =>
-                {
-                    result = enlistment.Authentication.TryInitialize(tracer, enlistment, out authError);
-                    return true;
-                },
-                "Authenticating");
-
-            authErrorMessage = authError;
-            return result;
+            // Don't show this in a spinner because the credential helper may need to show terminal prompts
+            this.Output.WriteLine("Authenticating...");
```

Comment:
Please update the "Example Workflow" section of `Readme.md` to match this change.